controllers/api_v1/keyword_test.go

```diff
@@ -127,6 +127,186 @@ func TestKeywordAPIControllerDbTestSuite(t *testing.T) {
 	suite.Run(t, new(KeywordAPIControllerDbTestSuite))
 }
 
+func (s *KeywordAPIControllerDbTestSuite) TestFetchKeywordsWithValidParams() {
+	keyword := models.Keyword{UserID: s.user.ID, Keyword: "testKeyword"}
+	db.GetDB().Create(&keyword)
+
+	headers := http.Header{}
+	headers.Set("Authorization", "Bearer test-access")
+
+	resp := testHttp.PerformRequest(s.engine, "GET", "/api/v1/keywords", headers, nil)
+	respBodyData, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Error(err)
+	}
+
+	var parsedRespBody map[string][]api_helper.DataResponseObject
+	err = json.Unmarshal(respBodyData, &parsedRespBody)
+	if err != nil {
+		log.Error(err)
+	}
```

Comment:
@gutakk Moving these to a helper function would be great, we can do it in another ticket 🙌🏼

Reply:
I decide to refactor it in the `test refactor` task for all related test codes.